src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px 0;
```

Comment:
You are adding both top and bottom margins to your `.form` class. It's recommended to be consistent with your margins and add only top or bottom, but not both, to avoid potential margin collapse. Choose one direction (for example, only bottom margins) and stick to it throughout your project.